paasta_tools/deployd/metrics.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import time
+
+from paasta_tools.deployd.common import PaastaThread
+
+try:
+    import yelp_meteorite
```

Comment:
I decided to follow @Rob-Johnson's advice in the end and turn it into an interface like our logging interfaces. That way we can still use yelp_meteorite but anyone that wanted to use statsd directly would have an easy way to add it in.